crates/test-macros/src/lib.rs

```diff
@@ -259,21 +231,37 @@ fn expand(test_config: &TestConfig, func: Fn) -> Result<TokenStream> {
             func_name.span(),
         );
 
-        let config_setup = test_config.wasm_features.iter().map(|f| {
-            let method_name = Ident::new(&format!("wasm_{f}"), f.span());
-            quote! {
-                config.#method_name(true);
-            }
-        });
+        let ignore = if strategy.should_fail(&test_config.flags) {
+            quote!(#[ignore])
```

Comment:
Maybe this should be `#[should_panic]`? I get that this keeps the original behavior but more just wondering: if we check for panics and then it stops panicking then we know more quickly which Winch tests can be enabled.

Reply:
Good point! Works well 👍